bpf/node_config.h

```diff
@@ -16,20 +16,23 @@
 DEFINE_MAC(NODE_MAC, 0xde, 0xad, 0xbe, 0xef, 0xc0, 0xde);
 #define NODE_MAC fetch_mac(NODE_MAC)
 
-DEFINE_IPV6(ROUTER_IP, 0xbe, 0xef, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0x0, 0x1, 0x0, 0x1, 0x0, 0x0);
+DEFINE_IPV6(ROUTER_IP, 0xbe, 0xef, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0x0,
+	    0x1, 0x0, 0x1, 0x0, 0x0);
 #define ENCAP_IFINDEX 1
 #define HOST_IFINDEX 1
 #define CILIUM_IFINDEX 1
 #define NATIVE_DEV_MAC_BY_IFINDEX(_) { .addr = { 0xce, 0x72, 0xa7, 0x03, 0x88, 0x56 } }
-DEFINE_IPV6(HOST_IP, 0xbe, 0xef, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0xa, 0x0, 0x2, 0xf, 0xff, 0xff);
+DEFINE_IPV6(HOST_IP, 0xbe, 0xef, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0xa,
+	    0x0, 0x2, 0xf, 0xff, 0xff);
 #define HOST_ID 1
 #define WORLD_ID 2
 #define UNMANAGED_ID 3
 #define HEALTH_ID 4
 #define INIT_ID 5
 #define REMOTE_NODE_ID 6
 #define HOST_IFINDEX_MAC { .addr = { 0xce, 0x72, 0xa7, 0x03, 0x88, 0x56 } }
-#define NAT46_PREFIX { .addr = { 0xbe, 0xef, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xa, 0x0, 0x0, 0x0, 0x0, 0x0 } }
+#define NAT46_PREFIX { .addr = { 0xbe, 0xef, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, \
+				 0x0, 0x0, 0xa, 0x0, 0x0, 0x0, 0x0, 0x0 } }
```

Comment:
Short line requirement is annoying here, makes the IP addresses less readable IMO :/

Reply:
Right, ok I'll revert those. I see some conflicts appeared anyway.